src/components/task-teacher-review/exercise.cjsx

```diff
@@ -60,21 +64,75 @@ TaskTeacherReviewExercise = React.createClass
     </BS.Accordion>
 
   render: ->
-    {answers, answered_count} = @props
-    question = @getQuestion()
+    {question, questionStats} = @props
+    {answers, answered_count} = questionStats
 
-    if ExerciseStore.hasQuestionWithFormat('free-response', {content: @props.content})
+    question.answers = @gatherAnswerStatsById()
+
+    if ExerciseStore.doesQuestionHaveFormat('free-response', question)
       studentResponses = if answers.length then @renderFreeResponse() else @renderNoFreeResponse()
 
-    <CardBody className='task-step openstax-exercise' pinned={false}>
-      <Question
-        model={question}
-        answered_count={answered_count}
-        type='teacher-review'
-        exercise_uid={@props.content.uid}
-        onChangeAttempt={@onChangeAnswerAttempt}>
-        {studentResponses}
-      </Question>
+    <Question
+      model={question}
+      answered_count={answered_count}
+      type='teacher-review'
+      onChangeAttempt={@onChangeAnswerAttempt}>
+      {studentResponses}
+    </Question>
+
+
+TaskTeacherReviewQuestionTracker = React.createClass
+  displayName: 'TaskTeacherReviewQuestionTracker'
+  mixins: [ScrollTrackerMixin]
+  render: ->
+    questionProps = _.pick(@props, 'question', 'questionStats')
+    <TaskTeacherReviewQuestion {...questionProps}/>
+
+
+TaskTeacherReviewExercise = React.createClass
+  displayName: 'TaskTeacherReviewExercise'
+  propTypes:
+    content: React.PropTypes.object.isRequired
+    question_stats: React.PropTypes.array.isRequired
+
+  getQuestionStatsById: (questionId) ->
+    {question_stats} = @props
+    questionId = questionId.toString()
+    _.findWhere(question_stats, question_id: questionId)
+
+  renderQuestion: (question, index) ->
+    questionStats = @getQuestionStatsById(question.id)
+    {setScrollPoint, unsetScrollPoint, scrollState} = @props
+    {key} = scrollState
+    scrollState = _.extend {}, scrollState, {key: key + index}
+
+    <TaskTeacherReviewQuestionTracker
+      key={"task-review-question-#{question.id}"}
+      question={question}
+      questionStats={questionStats}
+      scrollState={scrollState}
+      setScrollPoint={setScrollPoint}
+      unsetScrollPoint={unsetScrollPoint}/>
+
+  render: ->
+    {content} = @props
+    {questions, uid} = content
+
+    exercise = _.map questions, @renderQuestion
+    {stimulus_html} = content
+
+    stimulus = <ArbitraryHtmlAndMath
+      {...htmlAndMathProps}
+      className='exercise-stimulus'
```

Comment:
`htmlAndMathProps` is undefined, is causing the assignment metrics screen to fail 😿